test/integration/targets/seboolean/tasks/seboolean.yml

```diff
@@ -20,6 +20,16 @@
     name: policycoreutils-python
   when:
     - ansible_distribution == 'RedHat'
+    - ansible_distribution_major_version|int < 8
+
+- name: install requirements for rhel8 beta
```

Comment:
Unless we expect this to change after Beta, drop the beta reference.

Also, update the previous task to mention it's for RHEL 7 and earlier.